components/light/xiaomi.py

```diff
@@ -60,27 +39,23 @@ def __init__(self, device, name, xiaomi_hub):
 
         self._state = False
         self._rgb = (255,255,255)
-        self._ct = None
         self._brightness = 180
-        self._xy_color = (.5, .5)
-        self._white = 200
-        self._effect_list = None
-        self._effect = None
```

Comment:
The light does not support color_temp, xy_color, white_value or effects. Then it is no reason to add it.